### DISCUSSION TITLE
Wind Tunnel setup-FIST

### DISCUSSION THREAD

**Discussion by SaharRasoulpour** [firemodels/fds#13104] [Q&A]
Hello everyone,

I would appreciate your help with an issue I'm facing in my FDS input file for a wind tunnel model. Here are the details of my setup:

The box dimensions are 60x26x10.5.
The inlet has laminar air flow with a velocity of 1 m/s.
The left exit has an open boundary condition.
An infrared heater is modeled as a 9x9 cm surface, maintained at a temperature of 971 K, producing a heat flux of 50 kW/m² at the surface of the heater (located at the top of the box).
A few non-reacting fuel particles are held at a constant temperature of 1500°C, modeled as the ignitor. These are located 1.2 cm downstream of the fuel elements and centered 6 mm off the bottom.
After configuring my FDS input file, I encountered an error: ERROR (101): Problem with PART line.

Could you please guide me on how to resolve this issue with the PART line? Additionally, is there a specific example input file in FDS that I can refer to for a similar case?

<img width="524" alt="image" src="https://github.com/firemodels/fds/assets/143542094/361455dc-843b-4817-adc5-9ede301b76f6">
[Douglas_Fir_Needles.txt](https://github.com/user-attachments/files/16021591/Douglas_Fir_Needles.txt)

**Comment by drjfloyd** [comment]
That error means you have an error on the input being referred to. Either you have a keyword that doesn't belong on that type of input, a typographical error, the wrong type of data for the intput (for example, giving a string when a  number is needed).  Compare your input line to the table of permitted inputs in the User's Guide for the specific version of FDS you are using.

**Comment by SaharRasoulpour** [comment]
Thank you for your response. If we have a FIST configuration in WFDS, how should we rewrite the input file in the FDS format? Are there any resources about the conversion process and rules? Which parameters need to be changed? Your assistance would be greatly appreciated.

**Comment by mcgratta** [comment]
I do not know what a FIST configuration is, and WFDS is not supported anymore.

**Comment by SaharRasoulpour** [comment]
In the mass loss rate plot for this input file, I’m observing a strange plateau after several time steps, whereas the plot should show an increasing trend. Could you help me identify the issue in my input file?
[ouglas_Fir_Needles_FMC130.txt](https://github.com/user-attachments/files/16042128/ouglas_Fir_Needles_FMC130.txt)

**Comment by mcgratta** [comment]
```
&INIT PART_ID='IGNITOR ELEMENTS', XB=0.243,0.2505,0.1,0.19,0.000,0.005, DENSITY=1., TEMPERATURE= 1500. /
```
This line is not the way you assign a temperature to particles. See examples in the User's Guide chapter on particles.

**Comment by SaharRasoulpour** [comment]
After editing and adjusting some variables in my input file, I still encounter the same issue: my MLR plot becomes plateaued after a while. Could you clarify how I can resolve this problem?

Additionally, I have another question. When I specify T_end = 250 sec, my simulation finishes in 25 sec. Is there any reason for this discrepancy?

&HEAD CHID='Douglas_Fir_Needles_FMC70', TITLE='Wind Tunnel Setup' /

&MESH IJK=120,52,21, XB=0.0,0.3, 0.01,0.27, -0.015,0.09 /
&MESH IJK=120,52,21, XB=0.3,0.6, 0.01,0.27, -0.015,0.09 /

&MESH IJK=3,52,21, XB=0.00,0.015, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.015,0.03, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.03,0.045, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.045,0.06, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.06,0.075, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.075,0.09, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.09,0.105, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.105,0.12, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.12,0.135, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.135,0.15, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.15,0.165, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.165,0.18, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.18,0.195, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.195,0.21, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.21,0.225, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.225,0.24, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.24,0.255, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.255,0.27, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.27,0.285, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.285,0.30, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.30,0.315, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.315,0.33, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.33,0.345, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.345,0.36, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.36,0.375, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.375,0.39, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.39,0.405, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.405,0.42, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.42,0.435, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.435,0.45, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.45,0.465, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.465,0.48, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.48,0.495, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.495,0.51, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.51,0.525, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.525,0.54, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.54,0.555, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.555,0.57, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.57,0.585, 0.01,0.27, -0.015,0.09 /
&MESH IJK=3,52,21, XB=0.585,0.60, 0.01,0.27, -0.015,0.09 /

&REAC ID='WOOD', FUEL='METHANE', SOOT_YIELD=0.02, C=1.0, H=4.0, HEAT_OF_COMBUSTION=55500. /

&TIME T_END=250., DT=0.5/

&SPEC ID= 'METHANE'/
&SPEC ID='WATER VAPOR' /
&SPEC ID='OXYGEN' /
&SPEC ID='CARBON DIOXIDE' /

&OBST XB=0.0,0.6,0.01,0.01,0.0,0.09, COLOR='GRAY', TRANSPARENCY=0.5 /
&OBST XB=0.0,0.6,0.27,0.27,0.0,0.09, COLOR='GRAY', TRANSPARENCY=0.5 /

&OBST XB=0.0,0.6,0.01,0.1,0.09,0.09, COLOR='SILVER', TRANSPARENCY=1.0 /
&OBST XB=0.0,0.6,0.19,0.27,0.09,0.09, COLOR='SILVER', TRANSPARENCY=1.0 /
&OBST XB=0.0,0.255,0.1,0.19,0.09,0.09, COLOR='SILVER', TRANSPARENCY=1.0 /
&OBST XB=0.345,0.6,0.1,0.19,0.09,0.09, COLOR='SILVER', TRANSPARENCY=1.0 /

&OBST XB=0.0,0.6,0.01,0.1,0.0,0.0, COLOR='SILVER', TRANSPARENCY=1.0 /
&OBST XB=0.0,0.6,0.19,0.27,0.0,0.0, COLOR='SILVER', TRANSPARENCY=1.0 /
&OBST XB=0.0,0.255,0.1,0.19,0.0,0.0, COLOR='SILVER', TRANSPARENCY=1.0 /
&OBST XB=0.345,0.6,0.1,0.19,0.0,0.0, COLOR='SILVER', TRANSPARENCY=1.0 /

&OBST XB=0.255,0.345,0.1,0.19,-0.007,-0.007, COLOR='SILVER', TRANSPARENCY=1.0 /
&OBST XB=0.255,0.255,0.1,0.19,-0.007,0.0, COLOR='SILVER', TRANSPARENCY=1.0/
&OBST XB=0.345,0.345,0.1,0.19,-0.007,0.0, COLOR='SILVER', TRANSPARENCY=1.0 /
&OBST XB=0.255,0.345,0.1,0.1,-0.007,0.0, COLOR='SILVER', TRANSPARENCY=1.0 /
&OBST XB=0.255,0.345,0.19,0.19,-0.007,0.0, COLOR='SILVER', TRANSPARENCY=1.0 /

&OBST XB=0.243,0.2505,0.1,0.19,0.000,0.005, BULK_DENSITY=1. /
&OBST XB=0.255,0.345,0.1,0.19,-0.005,0.000, BULK_DENSITY=42.9 /

&SURF ID='AIR INLET', VEL=-1.0/
&VENT XB=0.6,0.6,0.01,0.27,0.0,0.09, SURF_ID='AIR INLET', COLOR='GRAY', TRANSPARENCY=0.5 /

&SURF ID='FIRE', TMP_FRONT=698.0 /

&SURF ID = 'IGNITOR ELEMENTS'
SURFACE_VOLUME_RATIO = 8366.
RAMP_T ='RAMP IGN'
LENGTH = 0.1
GEOMETRY = 'CYLINDRICAL' /
&PART ID='IGNITOR ELEMENTS', DRAG_COEFFICIENT=0.1, SURF_ID='IGNITOR ELEMENTS'
QUANTITIES='PARTICLE TEMPERATURE','PARTICLE MASS', STATIC=.TRUE., INITIAL_TEMPERATURE=1500. /
&RAMP ID='RAMP IGN', T=0.5, F=1.0 /
&RAMP ID='RAMP IGN', T=500, F=0.5 /
&INIT PART_ID='IGNITOR ELEMENTS', XB=0.243,0.2505,0.1,0.19,0.000,0.005, DENSITY=1./

&SURF ID = 'vegetation'
MATL_ID = 'dry pine'
MOISTURE_FRACTION = 0.7
SURFACE_VOLUME_RATIO = 8366.
LENGTH = 0.1
GEOMETRY = 'CYLINDRICAL' /

&MATL ID = 'dry pine'
DENSITY = 550.
CONDUCTIVITY = 0.1
SPECIFIC_HEAT= 1.0
N_REACTIONS = 1
REFERENCE_TEMPERATURE = 300.
NU_MATL = 0.25
NU_SPEC = 0.75
SPEC_ID = 'METHANE'
HEAT_OF_REACTION= 1000
MATL_ID = 'CHAR' /

&MATL ID = 'MOISTURE'
DENSITY = 1000.
CONDUCTIVITY = 0.1
SPECIFIC_HEAT= 4.184
N_REACTIONS = 1
REFERENCE_TEMPERATURE = 100.
NU_SPEC = 1.0
SPEC_ID = 'WATER VAPOR'
HEAT_OF_REACTION= 2500. /

&MATL ID = 'CHAR'
DENSITY = 200.
CONDUCTIVITY = 1.0
SPECIFIC_HEAT = 1.6 /

&PART ID='pine needles', DRAG_COEFFICIENT=0.375, SURF_ID='vegetation'
QUANTITIES='PARTICLE TEMPERATURE','PARTICLE MASS', STATIC=.TRUE., INITIAL_TEMPERATURE=20./

&INIT PART_ID='pine needles', XB=0.255,0.345,0.1,0.19,-0.005,0.0, DENSITY = 550./

&VENT XB = 0.6, 0.6, 0.0, 0.3, -0.015, 0.0, SURF_ID = 'OPEN' /
&VENT MB='XMIN',SURF_ID='OPEN' /
&VENT MB='YMIN',SURF_ID='OPEN' /
&VENT MB='YMAX',SURF_ID='OPEN' /

&VENT XB=0.255,0.345,0.1,0.19,0.09,0.09, SURF_ID='FIRE' /
&VENT XB=0.0,0.6,0.0,0.1,0.09,0.09, SURF_ID='OPEN' /
&VENT XB=0.0,0.6,0.19,0.3,0.09,0.09, SURF_ID='OPEN' /
&VENT XB=0.0,0.255,0.1,0.19,0.09,0.09, SURF_ID='OPEN' /
&VENT XB=0.345,0.6,0.1,0.19,0.09,0.09, SURF_ID='OPEN' /

&DUMP DT_SLCF = 2, DT_PART = 1 /
&DUMP PLOT3D_QUANTITY(1) = 'MASS FRACTION', PLOT3D_SPEC_ID(1) = 'METHANE' /
&DUMP PLOT3D_QUANTITY(2) = 'MASS FRACTION', PLOT3D_SPEC_ID(2) = 'WATER VAPOR' /
&DUMP PLOT3D_QUANTITY(3) = 'MASS FRACTION', PLOT3D_SPEC_ID(3) = 'OXYGEN' /
&DUMP PLOT3D_QUANTITY(4:5) = 'TEMPERATURE', 'VELOCITY', DT_PL3D = 5.0, WRITE_XYZ = .TRUE./

&SLCF PBY=0.13, QUANTITY='TEMPERATURE', VECTOR=.TRUE. /
&SLCF PBY=0.13, QUANTITY='VELOCITY', VECTOR=.TRUE. /
&SLCF PBY=0.13, QUANTITY='HRRPUV' /
&SLCF PBY=0.13, QUANTITY='MASS FRACTION', SPEC_ID='METHANE' /
&SLCF PBY=0.13, QUANTITY='MASS FRACTION', SPEC_ID='OXYGEN' /
&SLCF PBY=0.13, QUANTITY='MASS FRACTION', SPEC_ID='CARBON DIOXIDE' /
&SLCF PBY=0.13, QUANTITY='MASS FRACTION', SPEC_ID='WATER VAPOR' /

&SLCF PBZ=0.0025, QUANTITY='TEMPERATURE', VECTOR=.TRUE. /
&SLCF PBZ=0.0025, QUANTITY='VELOCITY', VECTOR=.TRUE. /
&SLCF PBZ=0.0025, QUANTITY='HRRPUV' /
&SLCF PBZ=0.0025, QUANTITY='MASS FRACTION', SPEC_ID='METHANE' /
&SLCF PBZ=0.0025, QUANTITY='MASS FRACTION', SPEC_ID='OXYGEN' /
&SLCF PBZ=0.0025, QUANTITY='MASS FRACTION', SPEC_ID='CARBON DIOXIDE' /
&SLCF PBZ=0.0025, QUANTITY='MASS FRACTION', SPEC_ID='WATER VAPOR' /

&DEVC XB=0.0,0.3, 0.01,0.27, -0.015,0.09, QUANTITY='MPUV', PART_ID='IGNITOR ELEMENTS', ID='mass IGNITOR', SPATIAL_STATISTIC='VOLUME INTEGRAL' /
&DEVC XB=0.3,0.6, 0.01,0.27, -0.015,0.09, QUANTITY='MPUV', PART_ID='IGNITOR ELEMENTS', ID='mass IGNITOR', SPATIAL_STATISTIC='VOLUME INTEGRAL' /
&DEVC XB=0.0,0.3, 0.01,0.27, -0.015,0.09, QUANTITY='MPUV', PART_ID='pine needles', ID='mass pine needles', SPATIAL_STATISTIC='VOLUME INTEGRAL' /
&DEVC XB=0.3,0.6, 0.01,0.27, -0.015,0.09, QUANTITY='MPUV', PART_ID='pine needles', ID='mass pine needles', SPATIAL_STATISTIC='VOLUME INTEGRAL' /

&DEVC XB=0.0,0.6, 0.01,0.27, -0.015,0.09, QUANTITY='MPUV', PART_ID='IGNITOR ELEMENTS', ID='mass IGNITOR', SPATIAL_STATISTIC='VOLUME INTEGRAL' /
&DEVC XB=0.0,0.6, 0.01,0.27, -0.015,0.09, QUANTITY='MPUV', PART_ID='pine needles', ID='mass IGNITOR', SPATIAL_STATISTIC='VOLUME INTEGRAL' /
&DEVC XB=0.0,0.6, 0.01,0.27, -0.015,0.09, QUANTITY='MPUV', PART_ID='pine needles', ID='solid mass', SPATIAL_STATISTIC='VOLUME INTEGRAL' /
&DEVC XB=0.0,0.6, 0.01,0.27, -0.015,0.09, QUANTITY='DENSITY', SPEC_ID='WATER VAPOR', ID='water vapor mass', SPATIAL_STATISTIC='VOLUME INTEGRAL' /
&DEVC XB=0.0,0.6, 0.01,0.27, -0.015,0.09, QUANTITY='DENSITY', SPEC_ID='METHANE', ID='METHANE mass', SPATIAL_STATISTIC='VOLUME INTEGRAL' /
&DEVC XB=0.0,0.6, 0.01,0.27, -0.015,0.09, QUANTITY='TEMPERATURE', ID='TEMPERATURE' /

&TAIL /

**Reply by obscureed** [reply]
I have not looked at this model in detail (and I am not associated with NIST), but one thing jumps out at me: You have two &MESH blocks to define the volume, but then you have 40 more &MESH blocks, each 3 cells wide in the x-direction, overlapping with the original two. I do not see any benefit in the 40 blocks. It is possible that they will be ignored, but I think they cannot help -- I would recommend rerunning without those 40 blocks. The two remaining blocks split the model in half, dividing the fire bed etc as well. I would prefer to have only one mesh block to contain the main interesting features in this model, perhaps with a second block for "earlier tunnel" and a third block for "later tunnel". This is probably not critical.

If your model has ended early, you could look in the .out file for explanation. (It could also be a hardware or operating-system effect.)

Some other thoughts:
• It might be helpful to run a smaller geometry at first, just to have quicker run-times while you are checking the setup.
• I do not see any benefit in declaring the side-walls (YMIN and YMAX) etc to be OPEN, and then blocking them with &OBST. It only makes it harder to see into your model.
• I have not worked out why you have a compartment in the negative z, separated from the upper compartment by a thin wall.
• I ran your model for a couple of simulated seconds, and tried to display particles in smokeview, but I got a message "***warning: particle files have no particles". It seems that this is worth checking.

Good luck!

**Reply by SaharRasoulpour** [reply]
Hello, thank you for your reply. I removed the 40 mesh block as well as the split model from my configuration. Instead, I reduced the mesh size to achieve a quicker response. Currently, I am rewriting a WFDS input file into FDS input file format. The negative Z direction was mentioned in the WFDS file.

I adjusted my reaction for methane using lumped components. For the infrared heater, modeled as a 9x9 cm surface at a temperature of 971 K producing a heat flux of 50 kW/m² on top of the domain, I set HRRPUA to 50000. However, I am still unsure if I need to import this value into my input file.

In other sections, I defined the ignitor and vegetation with pine needles that undergo pyrolysis with a specified moisture fraction. I have adjusted these parameters, but my total mass loss rate (MLR) and heat release rate (HRR) do not match the sample run with WFDS.

Regarding particles, do you mean I need to include the part ID in the ignitor and vegetation sections? My adjusted input file is:


&HEAD CHID='NEEDLE_FMC70', TITLE='Wind Tunnel Setup' /
&TIME T_END=250., DT=0.01, WALL_INCREMENT=1 /
&MISC TMPA=20./
&RADI RADIATION=T/
&MESH IJK=50,16,8, XB=0.0,0.6, 0.01,0.27,-0.015,0.09 /
&SPEC ID= 'METHANE'/
&SPEC ID='WATER VAPOR' /
&SPEC ID='CARBON DIOXIDE', LUMPED_COMPONENT_ONLY = .TRUE. /
&SPEC ID='SOOT', LUMPED_COMPONENT_ONLY = .TRUE. /
&SPEC ID='NITROGEN', LUMPED_COMPONENT_ONLY = .TRUE./
&SPEC ID='OXYGEN', LUMPED_COMPONENT_ONLY = .TRUE./
&SPEC ID = 'AIR', SPEC_ID = 'OXYGEN','NITROGEN', VOLUME_FRACTION = 1,3.761904762, BACKGROUND=.TRUE./
&SPEC ID = 'PRODUCT', SPEC_ID = 'CARBON DIOXIDE','WATER VAPOR','SOOT','NITROGEN',  VOLUME_FRACTION = 0.09379272627, 0.1914137271, 0.001914137271, 0.7128794094/

&REAC ID='WOOD', FUEL='METHANE', SOOT_YIELD=0.02, C=1.0, H=4.0, HEAT_OF_COMBUSTION=55500. 
      SPEC_ID_NU= 'METHANE', 'AIR' , 'PRODUCT'
      NU= -1, -1.98, 10.44857143/


&OBST XB=0.0,0.6,0.01,0.01,-0.015,0.09, COLOR='POWDER BLUE', TRANSPARENCY=0.3 /!FRONT
&OBST XB=0.0,0.6,0.27,0.27,-0.015,0.09, COLOR='POWDER BLUE', TRANSPARENCY=0.3 /!BACK

&OBST XB=0.0,0.6,0.01,0.1,0.09,0.09, COLOR='ROYAL BLUE', TRANSPARENCY=1.0 /!UP_FRONT
&OBST XB=0.0,0.6,0.19,0.27,0.09,0.09, COLOR='ROYAL BLUE', TRANSPARENCY=1.0  /!UP_BACK
&OBST XB=0.0,0.255,0.1,0.19,0.09,0.09, COLOR='ROYAL BLUE', TRANSPARENCY=1.0 /!UP_LEFT
&OBST XB=0.345,0.6,0.1,0.19,0.09,0.09, COLOR='ROYAL BLUE', TRANSPARENCY=1.0  /!UP_RIGHT

&OBST XB=0.0,0.6,0.01,0.1,-0.015,-0.015, COLOR='GRAY', TRANSPARENCY=1.0  /!BOTTOM_FRONT
&OBST XB=0.0,0.6,0.19,0.27,-0.015,-0.015, COLOR='GRAY', TRANSPARENCY=1.0  /!BOTTOM_BACK
&OBST XB=0.0,0.255,0.1,0.19,-0.015,-0.015, COLOR='GRAY', TRANSPARENCY=1.0  /!BOTTOM_LEFT
&OBST XB=0.345,0.6,0.1,0.19,-0.015,-0.015, COLOR='GRAY', TRANSPARENCY=1.0  /!BOTTOM_RIGHT


!VELICITY INLET:
&SURF ID='AIR INLET', VEL=-1.0, SPEC_ID='AIR'/
&VENT XB=0.6,0.6,0.01,0.27,-0.015,0.09, SURF_ID='AIR INLET', COLOR='ORCHID' /

!FIRE(TOP):
&SURF ID='FIRE', TMP_FRONT=698.0 , HRRPUA=50000,  COLOR='INDIAN RED'/ !box_of_fire_on_top
&VENT XB=0.255,0.345,0.1,0.19,0.09,0.09, SURF_ID='FIRE' /

!IGNITOR:
&SURF ID                        ='IGNITOR ELEMENTS'
        TMP_GAS_FRONT           = 1500.
        COLOR                   = 'BLACK'
        DRAG_COEFFICIENT        = 0.1
        RAMP_TMP_GAS_FRONT      ='RAMP IGN' /
&RAMP ID='RAMP IGN', T=0.5, F=1 /
&RAMP ID='RAMP IGN', T=500, F=0.5 /

!&INIT PART_ID='IGNITOR ELEMENTS', XB=0.243,0.2505,0.1,0.19,0.000,0.005, N_PARTICLES_PER_CELL=1, CELL_CENTERED=T, TEMPERATURE=1500./
&OBST XB=0.243,0.2505,0.1,0.19,0.000,0.005, COLOR='BLACK',  SURF_ID ='IGNITOR ELEMENTS'/   !box
&OBST XB=0.243,0.2505,0.1,0.19,0.000,0.000, COLOR='BLACK', SURF_ID ='IGNITOR ELEMENTS' / !botom
&OBST XB=0.243,0.2505,0.1,0.19,0.005,0.005, COLOR='BLACK', SURF_ID ='IGNITOR ELEMENTS' / !top
&OBST XB=0.243,0.2505,0.1,0.1,0.000,0.005, COLOR='BLACK', SURF_ID ='IGNITOR ELEMENTS' /  !front
&OBST XB=0.243,0.2505,0.19,0.19,0.000,0.005, COLOR='BLACK', SURF_ID ='IGNITOR ELEMENTS' / !back
&OBST XB=0.243,0.243,0.1,0.19,0.000,0.005, COLOR='BLACK', SURF_ID ='IGNITOR ELEMENTS' / !left
&OBST XB=0.2505,0.2505,0.1,0.19,0.000,0.005, COLOR='BLACK', SURF_ID ='IGNITOR ELEMENTS' / !right


!NEEDLE:
&PART ID                        ='pine needles'
       SURF_ID                  ='vegetation'
       DRAG_COEFFICIENT         = 0.375 
       QUANTITIES               ='PARTICLE TEMPERATURE'
       STATIC                   =.TRUE.
       INITIAL_TEMPERATURE      = 20./

&SURF ID                        = 'vegetation'
      MATL_ID                   = 'dry pine'
      MOISTURE_FRACTION         = 0.7
      SURFACE_VOLUME_RATIO      = 8366.
      COLOR			= 'GREEN',
      LENGTH			= 0.09
      RADIUS			= 0.00038
      GEOMETRY			= 'CYLINDRICAL'/

&MATL ID                        = 'dry pine'
      DENSITY                   = 550.
      CONDUCTIVITY              = 1.69E-1
      SPECIFIC_HEAT             = 1.8
      N_REACTIONS               = 1
      A                         = 3.63E4
      E                         = 60279.83355
      MATL_ID                   = 'CHAR'        
      NU_MATL                   = 0.25
      SPEC_ID                   = 'METHANE'
      NU_SPEC                   = 0.75
      HEAT_OF_REACTION          = 418/

&MATL ID                        = 'MOISTURE'
      DENSITY                   = 1000.
      CONDUCTIVITY              = 0.1
      SPECIFIC_HEAT             = 4.184
      N_REACTIONS               = 1
      A                         = 5.13E10
      E                         = 88000.24252
      NU_SPEC                   = 1.0
      SPEC_ID                   = 'WATER VAPOR'
      HEAT_OF_REACTION          = 2260. /

&MATL ID			= 'CHAR',
      DENSITY			= 300,
      CONDUCTIVITY		= 0.05,
      SPECIFIC_HEAT		= 0.42,
      E				= 74826,
      A				= 430.,
      NU_MATL			= 1,
      MATL_ID			= 'ASH',
      HEAT_OF_REACTION		= -12000/

&MATL ID			= 'ASH',
      DENSITY			= 67,
      CONDUCTIVITY		= 0.1,
      SPECIFIC_HEAT		= 1.244/
&INIT  ID='pine needles',   PART_ID='pine needles', XB=0.255,0.345,0.1,0.19,-0.015,0.0, DENSITY = 550./
&OBST XB=0.255,0.345,0.1,0.19,-0.015,0.000,COLOR='GREEN' BULK_DENSITY=42.9, SURF_ID='vegetation' / !box
&OBST XB=0.255,0.345,0.1,0.19,-0.015,-0.007, SURF_ID='vegetation', COLOR='GREEN'/ !FUELsurface(bottom)
&OBST XB=0.255,0.255,0.1,0.19,-0.015,0.0, SURF_ID='vegetation',COLOR='GREEN'/      !left
&OBST XB=0.345,0.345,0.1,0.19,-0.015,0.0,SURF_ID='vegetation', COLOR='GREEN' /     !right
&OBST XB=0.255,0.345,0.1,0.1,-0.015,0.0, SURF_ID='vegetation', COLOR='GREEN' /      !front
&OBST XB=0.255,0.345,0.19,0.19,-0.015,0.0,SURF_ID='vegetation', COLOR='GREEN'/    !back
!VENTS:
!&VENT XB = 0.6, 0.6, 0.0, 0.3, -0.015, 0.0, SURF_ID = 'OPEN' /
&VENT MB='XMIN',SURF_ID='OPEN' /
&VENT MB='YMIN',SURF_ID='OPEN' /
&VENT MB='YMAX',SURF_ID='OPEN' /
&VENT XB=0.0,0.6,0.0,0.1,0.09,0.09, SURF_ID='OPEN' /
&VENT XB=0.0,0.6,0.19,0.3,0.09,0.09, SURF_ID='OPEN' /
&VENT XB=0.0,0.255,0.1,0.19,0.09,0.09, SURF_ID='OPEN' /
&VENT XB=0.345,0.6,0.1,0.19,0.09,0.09, SURF_ID='OPEN' /
!DEVCES:

!TEMPERATURE:
&DEVC XB=0.0,0.6, 0.01,0.27, -0.015,0.09, QUANTITY='TEMPERATURE', ID='TEMPERATURE' /
!SPECS:
&DEVC XB=0.0,0.6, 0.01,0.27, -0.015,0.09, QUANTITY='DENSITY', SPEC_ID='WATER VAPOR', ID='MASS water vapor', SPATIAL_STATISTIC='VOLUME INTEGRAL'  /
&DEVC XB=0.0,0.6, 0.01,0.27, -0.015,0.09, QUANTITY='DENSITY', SPEC_ID='METHANE', ID='MASS METHANE', SPATIAL_STATISTIC='VOLUME INTEGRAL'  /
&DEVC XB=0.0,0.6, 0.01,0.27, -0.015,0.09, QUANTITY='DENSITY', SPEC_ID='WATER VAPOR', ID='MASS water vapor', SPATIAL_STATISTIC='MEAN'  /
&DEVC XB=0.0,0.6, 0.01,0.27, -0.015,0.09, QUANTITY='DENSITY', SPEC_ID='METHANE', ID='MASS METHANE', SPATIAL_STATISTIC='MEAN'  /
&DEVC XB=0.0,0.6, 0.01,0.27, -0.015,0.09, QUANTITY='DENSITY', SPEC_ID='SOOT', ID='MASS SOOT', SPATIAL_STATISTIC='MEAN'  /
&DEVC XB=0.0,0.6, 0.01,0.27, -0.015,0.09, QUANTITY='DENSITY', SPEC_ID='OXYGEN', ID='MASS OXYGEN', SPATIAL_STATISTIC='MEAN'  /
&DEVC XB=0.0,0.6, 0.01,0.27, -0.015,0.09, QUANTITY='DENSITY', SPEC_ID='NITROGEN', ID='MASS NITROGEN', SPATIAL_STATISTIC='MEAN'  /
&DEVC XB=0.0,0.6, 0.01,0.27, -0.015,0.09, QUANTITY='DENSITY', SPEC_ID='CARBON DIOXIDE', ID='MASS CARBON DIOXIDE', SPATIAL_STATISTIC='MEAN'  /

&DEVC XB=0.0,0.6, 0.01,0.27,-0.015,0.09, QUANTITY='MASS FRACTION', ID='MASS FRACTION CO2needle', SPEC_ID='CARBON DIOXIDE',  SPATIAL_STATISTIC='MEAN' /
&DEVC XB=0.0,0.6, 0.01,0.27,-0.015,0.09, QUANTITY='MASS FRACTION', ID='MASS FRACTION O2needle', SPEC_ID='OXYGEN',  SPATIAL_STATISTIC='MEAN' /
&DEVC XB=0.0,0.6, 0.01,0.27,-0.015,0.09, QUANTITY='MASS FRACTION', ID='MASS FRACTION N2needle', SPEC_ID='NITROGEN',  SPATIAL_STATISTIC='MEAN' /
&DEVC XB=0.0,0.6, 0.01,0.27,-0.015,0.09, QUANTITY='MASS FRACTION', ID='MASS FRACTION CH4needle', SPEC_ID='METHANE',  SPATIAL_STATISTIC='MEAN' /
&DEVC XB=0.0,0.6, 0.01,0.27,-0.015,0.09, QUANTITY='MASS FRACTION', ID='MASS FRACTION watervaoprneedle', SPEC_ID='WATER VAPOR',  SPATIAL_STATISTIC='MEAN' /
&DEVC XB=0.0,0.6, 0.01,0.27,-0.015,0.09, QUANTITY='MASS FRACTION', ID='MASS FRACTION sootneedle', SPEC_ID='SOOT',  SPATIAL_STATISTIC='MEAN' /
&DEVC XB=0.0,0.6, 0.01,0.27, -0.015,0.09, QUANTITY='NORMALIZED MASS', PART_ID='pine needles', ID='pine needles1 mass(environtment)', SPATIAL_STATISTIC='MEAN'/
&DEVC XB=0.255,0.345,0.1,0.19,-0.015,0.0,  QUANTITY='NORMALIZED MASS', PART_ID='pine needles', ID='pine needles2 mass(self)', SPATIAL_STATISTIC='MEAN'/








&TAIL /

![_s0000](https://github.com/user-attachments/assets/43aa2409-0f97-47da-8a41-48b5ef147772)
The plot that I am trying to figure out is:

![image](https://github.com/user-attachments/assets/7564ea3a-fd9d-43e6-acbd-1f6cb7528b6f)
